bolt/org_config.go

```diff
@@ -0,0 +1,193 @@
+package bolt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/bolt/internal"
+)
+
+// Ensure OrganizationConfigStore implements chronograf.OrganizationConfigStore.
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+// OrganizationConfigBucket is used to store chronograf organization configurations
+var OrganizationConfigBucket = []byte("OrganizationConfigV1")
+
+// OrganizationConfigStore uses bolt to store and retrieve organization configurations
+type OrganizationConfigStore struct {
+	client *Client
+}
+
+func (s *OrganizationConfigStore) Migrate(ctx context.Context) error {
+	return nil
+}
+
+// Get retrieves an OrganizationConfig from the store
+func (s *OrganizationConfigStore) Get(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
+	var cfg chronograf.OrganizationConfig
+
+	err := s.client.db.View(func(tx *bolt.Tx) error {
+		return s.get(ctx, tx, orgID, &cfg)
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &cfg, nil
+}
+
+func (s *OrganizationConfigStore) get(ctx context.Context, tx *bolt.Tx, orgID string, cfg *chronograf.OrganizationConfig) error {
+	v := tx.Bucket(OrganizationConfigBucket).Get([]byte(orgID))
```

Comment:
what happens if this returns an empty slice?

Reply:
👍